app/javascript/analytics/dashboard.js

```diff
@@ -9,9 +9,11 @@ function resetActive(activeButton) {
   for (let i = 0; i < buttons.length; i += 1) {
     const button = buttons[i];
     button.classList.remove('crayons-tabs__item--current');
+    button.setAttribute('aria-current', '');
```

Comment:
Refactor (non-blocking): Any reason, why we don't remove the attribute? Just asking as per your  note in the PR description. Same question for other spots in the code.

> all other tabs either have an empty aria-current value, or no aria-current attribute at all

Reply:
Yeah there's a mix in the code, mostly because when we need to set it in the erb files, our only option is to set it to an empty string instead of omitting it completely. Both `aria-current=""` and no `aria-current` attribute at all are treated the same by assistive technology though 🙂 

In this particular case we could just removeAttribute, and it's a little cleaner so I'll change that 👍